### DISCUSSION TITLE
Weierd Lighting Boundaries in `Turning Objects into Lights`

### DISCUSSION THREAD

**Discussion by Popaulol** [RayTracing/raytracing.github.io#1522] [Debugging]
I am currently in the chapter on adding Lights in Raytracing: The Next week, and i am seeing this weierd lighting boundaries, both with BVH being used and without it.

![image](https://github.com/RayTracing/raytracing.github.io/assets/56090021/c665fec7-afbf-4fe2-bb34-a1c56b24540a)

My Code: https://github.com/Popaulol/RustTracer

**Comment by Popaulol** [comment]
(The Light gray boundary comes from my ppm viewer)

**Comment by hollasch** [comment]
Looks like your front face is pointing in the opposite direction.

**Reply by Popaulol** [reply]
Shouldn't that not matter since the plane emits light on it's entire surface?

But I'll try that out once I get back on my PC
